### ISSUE TITLE
Problems with README example config

### ISSUE THREAD

**Issue by faximan** [abcxyz/github-metrics-aggregator#254]
### TL;DR

The provided variables do not match what Terraform is expecting.

### Environment Details

```markdown
I was running with terraform 1.5.4 on commit f1a8e93d6582be95bba447d00ef25babbabfd563 but I believe this is the same at head.
```

### Additional information

Directly copying the config at https://github.com/abcxyz/github-metrics-aggregator?tab=readme-ov-file#deploy-the-service I get the following errors:

* Required attribute "automation_service_account_member" not specified: An attribute named "automation_service_account_member" is required here
* Required attribute "bigquery_project_id" not specified: An attribute named "bigquery_project_id" is required here
* Unexpected attribute: An attribute named "big_query_project_id" is not expected here
* Unexpected attribute: An attribute named "leech_bucket_name" is not expected here

Additionally, I'd love for `automation_service_account_member` and `webhook_domains` to be explained what they should be set to, currently it is unclear what sort of pre-configuring I need to do for these two. At first, I left `webhook_domains` empty, but this caused a hard to decipher error at apply time:

```
  │ Error: Error creating ManagedSslCertificate: googleapi: Error 400: Invalid value for field 'resource.managed': ''. Managed certificate details must be specified if type = MANAGED., invalid
  │ 
  │   with module.github_metrics_aggregator.module.gclb[0].google_compute_managed_ssl_certificate.default,
  │   on .terraform/modules/github_metrics_aggregator.gclb/modules/gclb_cloud_run_backend/main.tf line 72, in resource "google_compute_managed_ssl_certificate" "default":
  │   72: resource "google_compute_managed_ssl_certificate" "default" {
  │ 
  ╵
````

**Comment by faximan** [comment]
Thank you!!
